modules/gapi/include/opencv2/gapi/s11n.hpp

```diff
@@ -126,53 +212,133 @@ GAPI_EXPORTS std::unique_ptr<IIStream> getInStream(const std::vector<char> &p);
 
 // OpenCV types ////////////////////////////////////////////////////////////////
 
+/** @brief This operator is used to serialize cv::Point into IOStream object.
+ *
+ * For instance, it can be used when overloading serialization routines for
+ * RMat's adapter or types inside GCompileArgs.
+ *
+ * Check overloads for more examples.
+ * @param os IOStream object to serialize and store cv::Point.
+ * @param pt cv::Point to serialize.
+ * @return IOStream object with serialized cv::Point inside.
+ * @see serialize IOStream
+ */
 GAPI_EXPORTS IOStream& operator<< (IOStream& os, const cv::Point &pt);
+/** @brief This operator is used to deserialize cv::Point from IIStream object.
+ *
+ * For instance, it can be used when overloading deserialization routines for
+ * RMat's adapter or types inside GCompileArgs.
+ *
+ * Check overloads for more examples.
+ * @param is IIStream object to deserialize cv::Point from.
+ * @param pt reference to cv::Point to deserialize into.
+ * @return IIStream object with deserialized and extracted cv::Point from.
+ * @see deserialize IIStream
+ */
```

Comment:
I'd keep these operators undocumented. OpenCV's `@overload` macro may be not a good fit here (as it says "it is provided for convenience"). Just keep it as is

Reply:
Removed the documentation on operators